spark-common/src/main/resources/com/k15t/spark/content_iframe_wrapper.vm

```diff
@@ -0,0 +1,27 @@
+## appending WithHtml to variable name instructs Velocity not to add html escaping (like & -> &amp;)
+## which would otherwise broke the js embedded this way pretty badly...
+#set($iframeResizerJsWithHtml = $iframeResizerJs)
+<script>
+    ${iframeResizerJsWithHtml}
+</script>
+<iframe id="${iframeId}" scrolling="no" style="width: 100%; border: none;" src="${iframeSrc}"></iframe>
+<script>
```

Comment:
Is this script somehow scoped to the iframe created above?

There might be situations where multiple such iframes exist in the browser window at the same time.